bug_change_status_page.php

```diff
@@ -158,7 +159,7 @@
 
 <?php
 	$t_current_resolution = $t_bug->resolution;
-	$t_bug_is_open = $t_current_resolution < $t_resolved;
+	$t_bug_is_open = $t_current_resolution < $t_resolution_fixed;
```

Comment:
> I suggest renaming $t_bug_is_open to $t_bug_resolution_is_fixed

Done

> so my expectation would be that closing a reopened issue would mark it as fixed.

Right, this would be good, unfortunately we have
```
$g_resolution_enum_string = '10:open,20:fixed,30:reopened,40:unable to duplicate,50:not fixable,60:duplicate,70:not a bug,80:suspended,90:wont fix';
```
instead of
```
$g_resolution_enum_string = '10:open,20:reopened,30:suspended,40:fixed,50:unable to duplicate,60:not fixable,70:duplicate,80:not a bug,90:wont fix';
```

Furthermore we have
```
/**
 * Threshold resolution which denotes that a bug has been resolved and
 * successfully fixed by developers. Resolutions above this threshold
 * and below $g_bug_resolution_not_fixed_threshold are considered to be
 * resolved successfully.
 * @global integer $g_bug_resolution_fixed_threshold
 */
$g_bug_resolution_fixed_threshold = FIXED;

/**
 * Threshold resolution which denotes that a bug has been resolved without
 * being successfully fixed by developers. Resolutions above this
 * threshold are considered to be resolved in an unsuccessful way.
 * @global integer $g_bug_resolution_not_fixed_threshold
 */
$g_bug_resolution_not_fixed_threshold = UNABLE_TO_DUPLICATE;
```

I think that documentation of $g_bug_resolution_fixed_threshold is wrong.
```
Resolutions above this threshold
```
 
I think it should be
 ```
Resolutions above and including this threshold
 ```
Do you agree?
 
I don't see how we could have a clean solution without 
- reordering resolutions (which would mean data migration) or
- being able to configure individual resolutions as fixed/not fixed instead of the two threshold options.
 
I think this is beyond the scope of this PR, which is just to prevent strange resolution changes when closing issues, caused by comparing status values with resolution values.